src/fides/api/common_exceptions.py

```diff
@@ -25,6 +25,14 @@ class TraversalError(FidesopsException):
     """Fidesops error with the names of all nodes that could not be reached."""
 
 
+class UnreachableNodesError(TraversalError):
+    """Fidesops error with the names of all nodes that could not be reached."""
```

Comment:
Minor "Fidesops" was the repo that used to process DSR's versus Fidesctl that did more database scanning and D&D. Now they're all unified under one "Fides" but there's still fidesops/fidesctl references still lingering